spec/services/calculators/dependant_allowance_calculator_spec.rb

```diff
@@ -141,50 +141,50 @@ module Calculators
       end
     end
 
-    # Changing these to be 2021 dates.
```

Comment:
It's hard to tell in the github view, but do we have a set of tests for each threshold file?  
I seem to remember this biting us when someone changed an old threshold file and the tests didn't fail.
I would suggest having a 2021 block, and add a 2022 block so we can test both thresholds

Reply:
no tests for each threshold file, just to straddle the most recent threhsold period. I will add tests for all dates

Reply:
I wouldn't necessarily worry about _all_ dates, but I would definitely add new years. 
I seem to remember that someone did a search and replace on a value and changed multiple threshold files and we didn't spot it! 😬

Reply:
fixed